architecture/service_catalog/aws_broker.adoc

```diff
@@ -37,7 +42,7 @@ Before working with this guide, familiarize yourself with the following terms:
 The _Service Catalog_ is the {product-title} component that finds and presents the services a user has access to. It also gives that user the ability to provision new instances of those services and provide a way to bind the provisioned services to existing applications.
 
 AWS Service Brokers:: 
```

Comment:
Suggest change 'AWS Service Brokers' to 'AWS Service Broker'

Reply:
+1